homeassistant/components/stream/hls.py

```diff
@@ -170,7 +172,7 @@ async def handle(self, request, stream, sequence):
             return web.HTTPNotFound()
         headers = {"Content-Type": "video/iso.segment"}
         return web.Response(
-            body=segment.moof_data,
+            body=b"".join([part.data for part in segment.parts]),
```

Comment:
Optional: Consider making this a member function on `Segment` since it shows up here, in recorder, and also in some tests.  (One thing i'm not sure about is if its worth combining with the init prefix also with an extra boolean parameter or not, and think maybe not...)

Reply:
`get_bytes` (the same function but also adding the init) is already added in the ll-hls PR, but I looked again and it's only used in one place, so I am not sure it is worth it there.
Since we do use this a few times here, I'll add it here (but without adding the init). We can see if it's worth removing later.